app/src/main/res/values/strings.xml

```diff
@@ -18,7 +18,10 @@
     <string name="channel_name_balance_increased">Balance increased</string>
     <string name="channel_name_threshold_reached">Threshold reached</string>
     <string name="invalid_ussd_code">Invalid USSD code entered</string>
+    <string name="invalid_subscription">Invalid Subscription, check default</string>
     <string name="ussd_code">USSD code</string>
+    <string name="subscription_id">Subscription</string>
+    <string name="subscription_summary">Change default sim card</string>
```

Comment:
This string doesn't seem to be used.

Reply:
I'll use it as description for subscription preference.